packages/core/src/interpreter.ts

```diff
@@ -439,6 +478,14 @@ export class Actor<
         `Only event objects may be sent to actors; use .send({ type: "${event}" }) instead`
       );
     }
+    if (!('__id' in event)) {
```

Comment:
The `__id` is a hacky workaround to ensure that the same event isn't inspected twice